README.md

```diff
@@ -558,6 +558,74 @@ With the configuration above, when you run `terragrunt apply`, Terragrunt will c
 terraform apply -lock-timeout=20m -var-file=terraform.tfvars -var-file=terraform-secret.tfvars
 ```
 
+#### Required and conditional var-files
+
+Sometime, it is required to include variables to terraform conditionally. The problem is that if we use
+-var-file=<file.tfvars>, <file.tfvars> must have to exist, otherwise, we get an error from terraform.
+
+For example, you may want to include files if they are available only.
```

Comment:
I changed the documentation. Is it possible for you to remove the trailing blanks for the README.md file? It is annoying to always disable my option to remove trailing blank when I edit this file.

Regards,
Jo